sources/packages/api/src/route-controllers/note/note.aest.controller.ts

```diff
@@ -55,13 +56,12 @@ export class NotesController extends BaseController {
    * @param noteType Note type(General|Restriction|System Actions|Program) which is passed to filter the notes.
```

Comment:
Methods like `transformToNoteDTO` should not be added to the shared `utilities` folder if they are DTO specific.